docs/root/start/distro/gloo.rst

```diff
@@ -0,0 +1,114 @@
+.. _install_gloo:
+
+Envoy as Ingress, API and Function Gateway for Kubernetes with Gloo
+===================================================================
+
+Kubernetes users often need to allow traffic to flow from and to the cluster, 
+and Envoy is great for that purpose.
+The open source project `Gloo <https://gloo.solo.io>`_, which is built on top 
+of Envoy, is designed for microservices, monolithts and also applications that
+might want to leverage function as a service. Gloo can decouple client APIs 
+from upstream APIs at the routing level. In a simplistic way, Gloo is a 
+great and easy to use tool to get traffic inside your Kubernetes cluster.
+
+Continue reading for more information on how to get started with Gloo.
+This should only take a few minutes
+
+Installing Gloo
+---------------
+
+For this installation, theare are 3 main pre-requisites:
+
+* **Kubernetes version**: Gloo requires version 1.8 or higher. `Minikube <https://kubernetes.io/docs/getting-started-guides/minikube/>`_
```

Comment:
Link doesn't match the version specified. I'd link to https://v1-8.docs.kubernetes.io/docs/getting-started-guides/minikube/.

Reply:
given that the latest is higher than 1.8, I'd rather have it pointing to latest.

Reply:
Fair enough.

Reply:
The link is redirecting though. I'd link to https://kubernetes.io/docs/setup/minikube/ directly.